Example/Seam3-Carthage-iOS/Seam3_Carthage_iOS.h

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
All of the changes to the files under the Example folder need to be removed from the PR.